service/service_test.go

```diff
@@ -62,6 +62,14 @@ func TestMain(m *testing.M) {
 	// run all migration up to the most active
 	if err := migration.Up(); err != nil {
 		logger.Error(consts.TestTag, "Failed to load active migration files:", err.Error())
+		// hack to reset DB to default settings with no entries
+		logger.Info(consts.TestTag, "Resetting migration")
+		if downErr := migration.Down(); downErr != nil {
+			logger.Fatal(consts.TestTag, "Failed to migrate down", err.Error())
+		}
+		if upErr := migration.Up(); upErr != nil {
+			logger.Fatal(consts.TestTag, "Failed to load active migration files:", err.Error())
+		}
```

Comment:
This is hack to reset PSQL DB for unit testing.

line 63 `if err := migration.Up(); err != nil {` will complain that there is no change needed.
We want to drop everything to prevent collisions thus dropping the migration, and migrating again.